homeassistant/components/climate/honeywell.py

```diff
@@ -75,7 +76,7 @@ def _setup_round(username, password, config, add_devices):
         zones = evo_api.temperatures(force_refresh=True)
         for i, zone in enumerate(zones):
             add_devices(
-                [RoundThermostat(evo_api, zone['id'], i == 0, away_temp)]
+                [RoundThermostat(hass, evo_api, zone['id'], i == 0, away_temp)]
```

Comment:
Call `add_devices` with a second argument `True`. This means that `update` will be called directly after adding the entity. Then you should remove the call to `update` from the `__init__` method of `RoundThermostat`.
```py
add_devices([RoundThermostat(evo_api, zone['id'], i == 0, away_temp)], True)
```